plugins/iam/src/index.mjs

```diff
@@ -54,9 +54,11 @@ const SSHPublicKeyId = { ...str, required, comment: 'ID of the SSH public key' }
 const TagKeys = { ...arr, required, comment: 'Array of tag keys' }
 const Tags = { ...arr, comment: 'List of tags to attach to the resource', ref: userGuide + 'id_tags.html' }
 const UserName = { ...str, required, comment: 'User name' }
-const valPaginate = { type: 'boolean', comment: 'Enable automatic result pagination; use this instead of making your own individual pagination requests' }
+const valPaginate = { type: [ 'boolean', 'string' ], comment: 'Enable automatic result pagination; use this instead of making your own individual pagination requests' }
 const VersionId = { ...str, required, comment: 'ID of the policy version; typically `v<n>`' }
 const VirtualMFADeviceName = { ...str, required, comment: 'Name of the virtual MFA device' }
+const valPaginateAsyncOnly = { type: 'string', comment: 'Set to `iterator` to enable async result pagination; response structure is not compatible with the fully automated paginator' }
```

Comment:
Minor copy tweak: `'Set to `iterator` to enable automatic result pagination via async iterator; use this instead of making your own individual pagination requests'`